etc/config-private.pl

```diff
@@ -165,6 +165,16 @@
     #     username => '...',
     #     password => '...',
     # );
+
+    # Configuration of BlobStore. This is the new storage abstraction used to
+    # store any blobs (images, userpics, media, etc) that need storage. For small
+    # sites/single servers, the localdisk mode is useful. For production
+    # systems S3 should be used.
+    # %BLOBSTORE = (
+    #     localdisk => {
+    #         path => "$LJ::HOME/var/blobstore",
+    #     },
+    # );
```

Comment:
I don't honestly remember why I created DW::PRIVATE, but it's only a small part of the config-private.pl configuration file. Some things in the LJ package reach into DW::PRIVATE. If there's a place to document what _should_ be in these files I'm all ears, btw, I can update.